HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentViewModel.swift

```diff
@@ -29,54 +30,94 @@ final class PaymentComponentViewModel {
+    }
+    private let payInvoiceViewBackgroundColorString: String?
+    private let defaultPayInvoiceViewBackgroundColor: UIColor = UIColor.GiniColors.accent1
+
+    // pay invoice view text color
     let payInvoiceLabelText: String = NSLocalizedStringPreferredFormat("ginihealth.paymentcomponent.payInvoice.label", comment: "")
-    let payInvoiceLabelAccentColor: UIColor
+    var payInvoiceLabelAccentColor: UIColor {
+        if let payInvoiceLabelAccentColorString, let textHexColor = payInvoiceLabelAccentColorString.toColor() {
+            return textHexColor
+        }
+        return defaultpayInvoiceLabelAccentColorString
+    }
+    private let payInvoiceLabelAccentColorString: String?
+    private let defaultpayInvoiceLabelAccentColorString: UIColor = UIColor.GiniColors.dark7
+
+    // pay invoice view font
     let payInvoiceLabelFont: UIFont
-    
+
+    var isPaymentProviderInstalled: Bool {
+        if let paymentProviderScheme, let url = URL(string: paymentProviderScheme), UIApplication.shared.canOpenURL(url) {
+            return true
+        }
+        return false
+    }
+    private var paymentProviderScheme: String?
+
     weak var delegate: PaymentComponentViewModelProtocol?
     
-    init(bankName: String,
-         bankIconName: String, 
-         payInvoiceAccentColor: GiniColor,
-         payInvoiceTextColor: GiniColor,
+    init(paymentProvider: PaymentProvider?,
          giniHealth: GiniHealth) {
         self.giniHealth = giniHealth
         self.moreInformationLabelFont = GiniHealthConfiguration.shared.customFont.with(weight: .regular,
-                                                                          size: 13,
-                                                                          style: .caption1)
+                                                                                       size: 13,
+                                                                                       style: .caption1)
         self.moreInformationLabelLinkFont = GiniHealthConfiguration.shared.customFont.with(weight: .bold,
-                                                                              size: 14,
-                                                                              style: .linkBold)
-        self.selectBankLabelFont = GiniHealthConfiguration.shared.customFont.with(weight: .medium,
-                                                                     size: 14,
-                                                                     style: .subtitle2)
-        self.bankImageIconName = bankIconName
-        self.bankNameLabelText = bankName
+                                                                                           size: 14,
+                                                                                           style: .linkBold)
+        self.selectYourBankLabelFont = GiniHealthConfiguration.shared.customFont.with(weight: .medium,
```

Comment:
[Clarification]: why we don't use here GiniHealthConfiguration.shared.textStyleFonts[.subtitle2] instead?